src/moments.jl

```diff
@@ -30,23 +32,32 @@ whereas it's `length(x)-1` in `Base.var`. The impact is that this is not a
 weighted estimate of the population variance based on the sample; it's the weighted
 variance of the sample.
 """
-function Base.var(v::RealArray, wv::WeightVec; mean=nothing)
-    mean == 0 ? Base.varm(v, wv, 0) :
-    mean == nothing ? varm(v, wv, Base.mean(v, wv)) :
-    varm(v, wv, mean)
+function Base.var(v::RealArray, wv::AbstractWeights; mean=nothing, corrected=true)
+    mean == 0 ? varm(v, wv, 0; corrected=corrected) :
+    mean == nothing ? varm(v, wv, Base.mean(v, wv); corrected=corrected) :
+    varm(v, wv, mean; corrected=corrected)
 end
 
 ## var along dim
 
-Base.varm!(R::AbstractArray, A::RealArray, wv::WeightVec, M::RealArray, dim::Int) =
-    scale!(_wsum_centralize!(R, @functorize(abs2), A, values(wv), M, dim, true), inv(sum(wv)))
+function Base.varm!(R::AbstractArray, A::RealArray, wv::AbstractWeights, M::RealArray, dim::Int; corrected=true)
+    scale!(
```

Comment:
Again, maybe a style guide would be helpful. The argument I've heard for spreading args (for long function calls) across multiple lines is that it minimizes the size of you diffs later on. I'm not saying that's the right approach, but given that there are multiple styles floating around and the julia style guide doesn't discuss them maybe it's worth clarifying.